.github/workflows/ci.yaml

```diff
@@ -13,27 +10,20 @@ permissions:
 
 jobs:
   typos-check:
-    name: spell-typos-check
+    name: Check typos
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Actions Repository
-        uses: actions/checkout@v3
-
-      - name: Check spelling with custom config file
-        uses: crate-ci/typos@v1.14.8
+      - uses: actions/checkout@v4
+      - uses: crate-ci/typos@v1.14.8
```

Comment:
suggestion to use `crate-ci/typos@master` here, some usages in: https://github.com/crate-ci/typos/blob/master/docs/github-action.md

Reply:
```suggestion
      - uses: crate-ci/typos@master
```